Update USA_PG_REG1-4.cup

ventus99 opened the pull request:
add Connecticut waypoints and tasks

<!--
Thank you very much for contributing! Please fill out the following
questions to make it easier for us to review your changes.
-->

# The purpose of this change

<!--
Please enter a summary of the changes.
-->

# The source of the data (for e.g. new frequencies)

<!--
Please provide direct URLs if possible.
-->

lordfolken reviewed (changes requested):
Please don't remove the quotes. They are required by the new standard document.

lordfolken reviewed (commented):
https://downloads.naviter.com/docs/SeeYou_CUP_file_format.pdf

ventus99 commented:
The document you linked says only fields with a comma require double quotes:

> Fields containing a comma must be embraced in double quotes (")

I didn't see any other requirements for double quotes.

ventus99 commented:
I have made the suggested changes

ventus99 commented:
Removing the "???" causes an error in XCSoar and does not display the task properly.  @lordfolken

lordfolken reviewed (changes requested):
It needs to be a waypoint name, one of the above. Neither "" nor ??? is a valid waypoint in the list. 

"All successive columns, separated by commas Each column represents one waypoint name. The waypoint
name must match the name field of one waypoint listed above the "----Related Tasks----" line.

ventus99 commented:
Okay, I have switched all the ??? to a turnpoint.  

You are right, the Racetrack=True should have been removed, my mistake.  Racetrack=True is a custom command used for Proving Grounds tasks that allows any turnpoint to be used as a start.  It is now removed.
